### DISCUSSION TITLE
Dropdown select with text filter

### DISCUSSION THREAD

**Discussion by salucien** [Megabit/Blazorise#2338] [Q&A] [answered]
Hi !
I couldn't find it so I'm asking here : 
Is there any way to get a select component with a text box included to search a specific value in the list ?

**Comment by stsrki** [comment, marked as answer]
I've just answered the same question :) 

Look here https://github.com/stsrki/Blazorise/issues/2336

**Reply by salucien** [reply]
Oh sorry I didn't see it... 
thank you !
